### PR TITLE
Dread: Remove requirement to have grapple to obtain Elun access pickup

### PR DESCRIPTION
**Pull request by Tyranisaur**

### CODE REVIEW

**Review by XenoWars — CHANGES_REQUESTED** [review]
Inaccurate. Grapple is required if you have not used the flipper. If it's simple enough to be trickless, you should change it to an Or listing either Grapple or (After Super Missile Rotatable and Shoot Super Missile) (something along that for the event name)

**Comment by Tyranisaur** [comment]
Right I suppose that accuracy is required for things like elevator rando, different starting locations etc. Otherwise you normally need to flip the rotatable to have access to the super locked door anyway.

Also, if you open the super locked door from behind, does that break the super lock even without supers?

**Comment by XenoWars** [comment]
> Also, if you open the super locked door from behind, does that break the super lock even without supers?

Well, the super lock is on both sides in rando. All door covers break on both sides when you break one of them.

**Comment by XenoWars** [comment]
Might be possible to store a speed if the flipper isnt used?

File: randovania/games/dread/json_data/Ghavoran.txt [hunk 1]
```diff
@@ -554,6 +554,8 @@ Extra - asset_id: collision_camera_007
   * Extra - right_shield_def: None
   > Door to Navigation Station
       After Ghavoran - Super Missile Rotatable
+  > Dock from Spider Magnet Elevator
+      Before Ghavoran - Super Missile Rotatable
```
Comment (XenoWars) [inline randovania/games/dread/json_data/Ghavoran.txt:558]: Remove this. You can only access it after the flipper has been used. There's also no point in even having it as you can't go through it anyway

File: CHANGELOG.md [hunk 2]
```diff
@@ -32,6 +32,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Changed: Some Grapple options to include Grapple Movement
 - Changed: Some Movement tricks to Climb Sloped Tunnels
 - Changed: Some Movement tricks to Skip Cross Bomb
+- Changed: Allow pickup in Ghavoran Elun Transport Access by charging speed via navigation room
+- Changed: Allow traversal through the bottom of Ghavoran Flipper Room without flipping the spinner
```
Comment (XenoWars) [inline CHANGELOG.md:36]: This is already in the logic, so take this out.

**Review by XenoWars — CHANGES_REQUESTED** [review]

**Comment by XenoWars** [comment]
Probably just capitalize morph ball in the changelog and it's good.

**Review by XenoWars — APPROVED** [review]

**Pull request merged**